Gems/Atom/RHI/Code/Source/RHI/FrameGraphCompiler.cpp

```diff
@@ -700,7 +702,11 @@ namespace AZ::RHI
         {
             // First pass to calculate size needed.
             processCommands(TransientAttachmentPoolCompileFlags::GatherStatistics | TransientAttachmentPoolCompileFlags::DontAllocateResources);
-            memoryUsage = transientAttachmentPool.GetStatistics().m_reservedMemory;
+            auto statistics = transientAttachmentPool.GetStatistics();
+            for (auto& [deviceIndex, deviceStatistics] : statistics)
+            {
+                memoryUsage = deviceStatistics.m_reservedMemory;
```

Comment:
Is this going to be changed to account for all devices?

Reply:
yes